packages/web-commons/src/sanitizeHtml.ts

```diff
@@ -0,0 +1,15 @@
+import type { IFilterXSSOptions } from "xss";
+import xss from "xss";
+
+const defaultParams: IFilterXSSOptions = {
+  stripIgnoreTagBody: ["script"],
+  whiteList: {
+    div: ["style"],
+    span: ["style"],
+  },
+  css: true,
+};
+
+export const sanitizeHtml = (html: string, params: IFilterXSSOptions = defaultParams) => {
```

Comment:
It's better not to use `IFilterXSSOptions` directly. Keep only a single `html` parameter for now, we add parameters, if they'll be required